README.md

```diff
@@ -1 +1,197 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Glossary/HTML5">Html5</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">None</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">None</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive]**
+- **[Dynamic]**
+- **[Professional]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+none
+
+### Setup
+Clone this repository to your desired folder:
+
+### Install
+
+none
+
+### Usage
+
+Clone it
+
+### Run tests 
+none
+
+### Deployment
+none
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Than Trong Khanh**
+
+- GitHub: [khanh-devos](https://github.com/khanh-devos)
+- Twitter: [@khanhror](https://twitter.com/home?lang=en)
+- LinkedIn: [Khanh RoR](https://www.linkedin.com/in/khanh-than-trong-3b4344246/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[add some specific content to it]**
+- [ ] **[implement responsive]**
+- [ ] **[use javascript to add some dynamic feature]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, can invite me a cup of tea
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+If you like this project, can invite me a cup of tea
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[what are linters for?]**
+
+  - [A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipeia](https://www.linkedin.com/in/khanh-than-trong-3b4344246/)).]
+
+- **[Linters can be used for all programming languages ?]**
+
+  - [No, not all languages can use linters, [details](https://github.com/microverseinc/linters-config)]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] To avoid the 404 error, please 🙏🏽 add the LICENSE file to your repo and ensure that the path is the correct 👉🏽 `./LICENSE` for professional documentation. You can use these [instructions](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).